src/index.js

```diff
@@ -40,19 +42,26 @@ function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customD
     customDiffConfig,
   });
   let pass = true;
-  if (result.updated) {
+  let message = '';
+  if (result.result === ResultTypes.UPDATE) {
     // once transition away from jasmine is done this will be a lot more elegant and pure
     // https://github.com/facebook/jest/pull/3668
     snapshotState = updateSnapshotState(snapshotState, { updated: snapshotState.updated += 1 });
-  } else if (result.added) {
+  } else if (result.result === ResultTypes.ADD) {
     snapshotState = updateSnapshotState(snapshotState, { added: snapshotState.added += 1 });
     // see https://github.com/yahoo/blink-diff/blob/master/index.js#L251-L285 for result codes
-  } else if (result.code === 0 || result.code === 1) {
-    pass = false;
-  }
+  } else {
+    pass = result.result === ResultTypes.PASS;
+
+    if (!pass) {
+      const dp = parseInt(result.differencePercentage * 100, 10);
 
-  const message = () => 'Expected image to match or be a close match to snapshot.\n'
-                  + `${chalk.bold.red('See diff for details:')} ${chalk.red(result.diffOutputPath)}`;
+      message = `Expected image to match or be a close match to snapshot. ${dp}% different.\n`
```

Comment:
Can you keep this as a function and keep it as a `const`? Else it wont work in jest 21